specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -92,38 +91,38 @@ objects.
 
 === Transaction Requirements
 
-The Java EE platform must meet the following
+The Jakarta EE platform must meet the following
 requirements:
 
-* The Java EE platform must provide an object
+* The Jakarta EE platform must provide an object
 implementing the _javax.transaction.UserTransaction_ interface to all
 web components. The platform must publish the _UserTransaction_ object
 in the Java™ Naming and Directory Interface (JNDI) name space available
-to web components under the name _java:comp/UserTransaction_ .
-* The Java EE platform must provide classes
-that implement the transactional interceptors, as defined by the JTA
+to web components under the name java:comp/UserTransaction_ .
```

Comment:
I wonder what this trailing "_" is about? maybe it should be "_java" still to do something like emphasis?

Reply:
Correct, @tomjenkinson.  He missed the underscore before the "_java"...  Thanks!